ls1intum/Artemis
# [Bugfix] Fixing documentation

### Checklist

### Motivation and Context


The [Programming Exercises Access Matrix](https://artemis-platform.readthedocs.io/en/latest/admin/accessRights/#programming-exercises) is currently broken in the documentation and therefore not displayed.

### Description

Fixing the `Programming Exercises Access Matrix` and adding a `Lecture Access Matrix`.

### Steps for Testing

Once you have installed Sphinx you can locally create the documentation by executing `make html` in the `docs` folder.

1. Check whether the documentation is displayed properly.
2. Check whether the access rights that are specified are correct / double-check access rights that you doubt from the experience with Artemis

### Screenshots


- Broken Porgramming Exercise Matrix: 
![image](https://user-images.githubusercontent.com/63976129/132585164-b074f42c-e088-4e18-887c-433ac030dcd4.png)

- Fixed Programming Exercise Matrix:
![image](https://user-images.githubusercontent.com/63976129/132589017-647a503e-2e3b-40df-b82c-ada589951f8d.png)

- Added Lectures Matrix:
![image](https://user-images.githubusercontent.com/63976129/132589072-ab05421b-e304-4268-9eed-b5b00d80027b.png)

### Review Progress
- Code Review
- Manual Tests
